ddl/schematracker/checker.go

```diff
@@ -210,6 +210,12 @@ func (d Checker) DropSchema(ctx sessionctx.Context, stmt *ast.DropDatabaseStmt)
 	return nil
 }
 
+// RecoverSchema implements the DDL interface.
+func (d Checker) RecoverSchema(ctx sessionctx.Context, recoverSchemaInfo *ddl.RecoverSchemaInfo) (err error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
@lance6716 Could you give some descriptions/suggestions on the semantic of `RecoverSchema` of `Checker`/`SchemaTracker`?

I can see that some tools integration tests failed because of this: https://github.com/pingcap/tidb/runs/8089302161

Reply:
Can return nil. Because SchemaTracker is used to track MySQL DDL, and MySQL will not generate RecoverSchema.

Reply:
Dumpling tests have started to fail several days ago, not related to `Checker`/`SchemaTracker`

Reply:
> Dumpling tests have started to fail several days ago, not related to `Checker`/`SchemaTracker`

Got it, thanks for the update.